Monika After Story/game/script-topics.rpy

```diff
@@ -4703,6 +4703,71 @@ label monika_write:
     m "Thanks for listening~"
     return
     
+init 5 python:
+    for key in ['Psychological Quiz']:
+        monika_topics.setdefault(key,[])
+        monika_topics[key].append('Monika_Psychological_Quiz')
+    monika_random_topics.append('Monika_Psychological_Quiz')
+
+label Monika_Psychological_Quiz:
+    python:
+        cat_number = none
+    m "Hey [player]."
+    m "So I've been looking into some experiments online as of late and I came across a good one"
+    menu:
+        m "Do you want to try 1 or 2?"
+        "1":
+            m "You are in a field of flowers."
+            m "The owner has told you that you may take as many as you would like home with you."
+            menu:
+                m "How many do you take?"
+                "Just a few."
```

Comment:
`menu` block choices need a colon `:` after the choice. Add colons to your `menu` block choices in this block.

Reply:
I just noticed you have several other locations where this happens. Please make sure to add colons after a `menu` block choice.